https://github.com/limo-app/limo/pull/137

From a95b98c138cb7f8ee1082a14c1295c8c9173435d Mon Sep 17 00:00:00 2001
From: Alfred Wingate <parona@protonmail.com>
Date: Wed, 26 Feb 2025 15:47:19 +0200
Subject: [PATCH] Include implicit header for std::ranges::iota

GCC-15 stdlibc++ has changed its header includes meanings this doesn't
get implicitly included anymore.

https://en.cppreference.com/w/cpp/algorithm/ranges/iota

Signed-off-by: Alfred Wingate <parona@protonmail.com>
--- a/src/core/reversedeployer.cpp
+++ b/src/core/reversedeployer.cpp
@@ -6,6 +6,7 @@
 #include <format>
 #include <fstream>
 #include <iostream>
+#include <numeric>
 #include <ranges>
 
 namespace sfs = std::filesystem;
-- 
2.48.1
